test/sslapitest.c

```diff
@@ -1819,6 +1819,9 @@ static int execute_cleanse_plaintext(const SSL_METHOD *smeth,
 
     return testresult;
 }
+#endif /* !defined(OPENSSL_NO_TLS1_2) || !defined(OSSL_NO_USABLE_TLS1_3) \
+        * || !defined(OPENSSL_NO_DTLS)
+        */
```

Comment:
The comment after the `#endif` is a bit weird - there's a line continuation `\` on the first line, but not on the second. If it's kept to exactly match the `#if` line, maybe the close comment `*/` would be better at the end of line 1823?

Reply:
Closing the comment on the same line would not be consistent with our multi-line comment style. Although, actually the start of this multi-line comment is already not consistent with our style. I fixed it to be consistent with our style. The continuation line was there to match the `#if` - but it doesn't really matter. I removed it.